edb/server/bootstrap.py

```diff
@@ -908,22 +908,23 @@ def prepare_patch(
 
         metadata_user_schema = reflschema
 
-    elif kind.startswith('edgeql+user_ext'):
+    elif kind.startswith('edgeql+user'):
         assert '+schema' not in kind
```

Comment:
Maybe limit this a bit, this currently also supports `edgeql+user_etx`, which we probably want to catch before releasing.